bitnami/kong/README.md

```diff
@@ -0,0 +1,326 @@
+| `service.proxyHttpsNodePort`                     | Port to bind to for NodePort service type (proxy HTTPS)                                                | `nil`                                                   |
+| `service.adminHttpNodePort`                     | Port to bind to for NodePort service type (admin HTTP)                                                | `nil`                                                   |
+| `service.aminHttpsNodePort`                     | Port to bind to for NodePort service type (proxy HTTP)                                                | `nil`                                                   |
+| `service.annotations`                  | Annotations for kong service                                                                        | `{}`                                                    |
+| `service.loadBalancerIP`               | loadBalancerIP if kong service type is `LoadBalancer`                                               | `nil`                                                   |
+| `ingress.enabled`                      | Enable the use of the ingress controller to access the web UI                                          | `false`                                                 |
+| `ingress.certManager`                  | Add annotations for cert-manager                                                                       | `false`                                                 |
+| `ingress.annotations`                  | Annotations for the kong Ingress                                                                    | `{}`                                                    |
+| `ingress.hosts[0].name`                | Hostname to your kong installation                                                                  | `kong.local`                                         |
+| `ingress.hosts[0].paths`               | Path within the url structure                                                                          | `["/"]`                                                 |
+| `ingress.hosts[0].tls`                 | Utilize TLS backend in ingress                                                                         | `false`                                                 |
+| `ingress.hosts[0].tlsHosts`            | Array of TLS hosts for ingress record (defaults to `ingress.hosts[0].name` if `nil`)                   | `nil`                                                   |
+| `ingress.hosts[0].servicePort`            | Which service port inside the kong service to redirect                   | `nil`                                                   |
+| `ingress.hosts[0].tlsSecret`           | TLS Secret (certificates)                                                                              | `kong.local-tls`                                     |
```

Comment:
Service and Ingress parameters shouldn't be listed under "Deployment Parameters" section but "Exposure parameters" instead. Check KAFKA

Reply:
I will use the word Traffic Exposure instead, I've seen it in the Kubernetes documentation. The word Exposure just by itself has a different nuance to me (in terms of security).